src/centos/scripts/start-marklogic.sh

```diff
@@ -210,7 +210,7 @@ fi
 ################################################################
 # check marklogic init (eg. MARKLOGIC_INIT is set)
 ################################################################
-if [[ -f /var/opt/MarkLogic/DOCKER_INIT ]]; then
```

Comment:
Why do we change the flag from /var/opt/MarkLogic to /opt/MarkLogic? After this change we will lost the flag after the container restarts

Reply:
Sorry, this change got included as the branch was not up to date with develop. I will revert this change in this PR.